lint/src/main/java/org/simple/clinic/lint/ImageSrcDetector.kt

```diff
@@ -0,0 +1,53 @@
+package org.simple.clinic.lint
+
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.Implementation
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.ResourceXmlDetector
+import com.android.tools.lint.detector.api.Scope
+import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.XmlContext
+import org.w3c.dom.Element
+
+@Suppress("UnstableApiUsage")
+class ImageSrcDetector : ResourceXmlDetector() {
+
+  companion object {
+    const val ANDROID_URI = "http://schemas.android.com/apk/res/android"
+    const val IMAGE_VIEW = "ImageView"
+    const val ATTR_SRC = "src"
+
+    private const val ImageSrcIssueId = "ImageSrcIssueId"
+    private const val ImageSrcIssueDescription = "Please use `app:srcCompat` when setting an image resource"
+
+    val ImageSrcIssue = Issue.create(
+        id = ImageSrcIssueId,
+        briefDescription = ImageSrcIssueDescription,
+        explanation = ImageSrcIssueDescription,
```

Comment:
We've named it `ImageSrcIssueDescription` but we are describing how to solve the issue.

`briefDescription` should summarises the issue while `explanation` should also include on how the new lint rule will be solved.

Reply:
Ah, good point. Will change it